notebooks/Fine_tuning_Code_LLM_on_single_GPU.ipynb

```diff
@@ -0,0 +1,2338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    base_model.cuda()
</pre><p> </p><p>not: We can avoid this added LoC if we do <code>.cuda()</code> in the above line:</p><p> </p><pre class="ql-syntax" spellcheck="false">base_model = AutoModelForCausalLM.from_pretrained(
&nbsp; &nbsp; MODEL,
&nbsp; &nbsp; quantization_config=None,
&nbsp; &nbsp; device_map=None,
&nbsp; &nbsp; trust_remote_code=True,
&nbsp; &nbsp; torch_dtype=torch.bfloat16,
).cuda()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>